src/js/tech/html5.js

```diff
@@ -73,8 +73,10 @@ class Html5 extends Tech {
     // Our goal should be to get the custom controls on mobile solid everywhere
     // so we can remove this all together. Right now this will block custom
     // controls on touch enabled laptops like the Chrome Pixel
-    if (browser.TOUCH_ENABLED && options.nativeControlsForTouch === true) {
-      this.trigger('usenativecontrols');
+    if ((browser.TOUCH_ENABLED && options.nativeControlsForTouch === true) || browser.IS_IPHONE) {
+      this.setTimeout(function() {
```

Comment:
Got it, so we create the tech and then add the usenativecontrols listener.

"Native controls" to the player is really just controls: true/false to the tech. In player.loadTech could we check `player.techGet('controls')` and if it's true assume we're using native controls? Does that work in our favor with the iPhone controls-always-true issue?